src/openrct2/world/TileInspector.cpp

```diff
@@ -1065,4 +1070,4 @@ namespace OpenRCT2::TileInspector
     {
         return _highlightedElement == elem;
     }
-} // namespace OpenRCT2::TileInspector
+} // namespace OpenRCT2::TileInspector
```

Comment:
Unrelated change, please revert:
```suggestion
} // namespace OpenRCT2::TileInspector

```